packages/node_modules/@ciscospark/widget-message-meet/src/container.js

```diff
@@ -196,7 +196,7 @@ const injectedPropTypes = {
   fetchToUser: PropTypes.func.isRequired,
   setActivityMenuVisible: PropTypes.func.isRequired,
   toggleActivityMenu: PropTypes.func.isRequired,
-  spark: PropTypes.object.isRequired,
+  spark: PropTypes.object,
```

Comment:
Yea when the component first loads spark isn't guaranteed to be present so we actually get a propType warning.